docker-compose.yml

```
@@ -344,7 +344,7 @@ services:
           - s3.docker.mender.io
 
   mongo:
-    image: mongo:7.0
+    image: mongo:8.0
```

```suggestion
    image: mongo:${MONGO_VERSION:-8.0}
```
that would help me a bit. but I leave it up to you.